phc/services/analytics.py

```diff
@@ -133,3 +144,261 @@ def get_data_lake_query(self, query_id: str) -> ApiResponse:
             The data lake query get response
         """
         return self._api_call("analytics/query/%s" % query_id, http_verb="GET")
+
+    def list_data_lake_schemas(self, dataset_id: str) -> ApiResponse:
```

Comment:
We've been trying to use `project_id` as the public facing sdk name across the different resources.

Reply:
Done